std/traits.d

```diff
@@ -996,11 +996,11 @@ enum ParameterStorageClass : uint
      * class.
      */
     none    = 0,
-    scope_  = 0b000_1,  /// ditto
-    out_    = 0b001_0,  /// ditto
-    ref_    = 0b010_0,  /// ditto
-    lazy_   = 0b100_0,  /// ditto
-    return_ = 0b1000_0, /// ditto
+    scope_  = 1,    /// ditto
+    out_    = 2,    /// ditto
+    ref_    = 4,    /// ditto
+    lazy_   = 8,    /// ditto
+    return_ = 0x10, /// ditto
```

Comment:
It doesn't need to have leading zeros. Another possibility would be:

```D
   1,
   2,
   4,
   8,
0x10,
```
What I like about the alignment is when you compare what bit positions are set from multiple values, they are easier to compare.

I also think the binary is quite useless, and prefer hex. It took me a while when I started, but learning what all the hex values are is not that bad.